spec/util/subscribeToResult-spec.ts

```diff
@@ -0,0 +1,156 @@
+import {expect} from 'chai';
+import * as Rx from '../../dist/cjs/Rx';
+import {subscribeToResult} from '../../dist/cjs/util/subscribeToResult';
+import {OuterSubscriber} from '../../dist/cjs/OuterSubscriber';
+import {$$iterator} from '../../dist/cjs/symbol/iterator';
+import $$symbolObservable from 'symbol-observable';
+
+describe('subscribeToResult', () => {
+  it('should immeidately complete when subscribe to scalarObservable', (done: MochaDone) => {
```

Comment:
"immeidately" (which is a typo, lol), should be "synchronously", to avoid ambiguity.